nix-action.yml.mustache

```diff
@@ -0,0 +1,37 @@
+name: Nix CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
      - {{branch}}{{^branch}}master{{/branch}}
```
This is needed to work with GitHub's new conventions for the primary branch name. Should probably also be used in `docker-action.yml.mustache `, but GitHub doesn't allow me to do a suggestion for the code there

Reply:
Thanks, updated!